src/auslib/web/admin/swagger/api_v2.yml

```diff
@@ -463,8 +478,9 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ReleaseModifyResponse'
-                $ref: '#/components/schemas/ReleasePinNotSetResponse'
+                anyOf:
+                  - $ref: '#/components/schemas/ReleaseModifyResponse'
+                  - $ref: '#/components/schemas/ReleasePinNotSetResponse'
```

Comment:
This is an unrelated fix, I assume?

Reply:
Yup, a nit-picky one at that. The Swagger parser is complaining that `Map keys must be unique` I try to load the API spec in the Swagger editor. I opted for `anyOf` because it satisfies the test cases for this end-point